src/config.py.in

```diff
@@ -2,9 +2,10 @@
 #
 # eos-google-chrome-installer: helper script to install Google Chrome
 #
-# Copyright (C) 2016 Endless Mobile, Inc.
+# Copyright (C) 2017 Endless Mobile, Inc.
```

Comment:
Right, I have doubts here because the whole file is different now, but yes... I'll append